### PR TITLE
Add script music autoplay(javascript), add social media icon

### PR DESCRIPTION
**Pull request by irvanlabs**
1. Add script
2. Add social media icon (html, css)

### CODE REVIEW

File: app/Views/templates/default.php [hunk 1]
```diff
@@ -228,6 +231,24 @@ class="text-decoration-none">
...
+         if(!isMP3HasPlayed) {
+             document.querySelector('audio').play().then((x)=>console.info('sucess', x)).catch((e)=>console.error('fail', e))
+             played = true
+             document.onclick = null
+         }
+     }
+     document.ontouchstart = (event) => {
+         if(!isMP3HasPlayed) {
+             document.querySelector('audio').play().then((x)=>console.info('sucess', x)).catch((e)=>console.error('fail', e))
+             played = true
```
Comment (bokunodev) [inline app/Views/templates/default.php:247]: variablenya `hasPlayed` bukan played

File: app/Views/templates/default.php [hunk 2]
```diff
@@ -228,6 +231,24 @@ class="text-decoration-none">
       integrity="sha512-bPs7Ae6pVvhOSiIcyUClR7/q2OAsRiovw4vAkX+zJbw3ShAeeqezq50RIIcIURq7Oa20rW2n2q+fyXBNcU9lrw=="
       crossorigin="anonymous" referrerpolicy="no-referrer"></script>
    <script type="text/javascript" src="/js/script.js"></script>
+   <script type="text/javascript">
+
+     let isMP3HasPlayed = false
+     document.onclick = (event) => {
+         if(!isMP3HasPlayed) {
+             document.querySelector('audio').play().then((x)=>console.info('sucess', x)).catch((e)=>console.error('fail', e))
+             played = true
```
Comment (bokunodev) [inline app/Views/templates/default.php:240]: variablenya `hasPlayed` bukan played

**Review by bokunodev — CHANGES_REQUESTED** [review]
variablenya `hasPlayed` bukan played

**Comment by irvanlabs** [comment]
Done

File: app/Views/templates/default.php [hunk 3]
```diff
@@ -228,6 +231,24 @@ class="text-decoration-none">
       integrity="sha512-bPs7Ae6pVvhOSiIcyUClR7/q2OAsRiovw4vAkX+zJbw3ShAeeqezq50RIIcIURq7Oa20rW2n2q+fyXBNcU9lrw=="
       crossorigin="anonymous" referrerpolicy="no-referrer"></script>
    <script type="text/javascript" src="/js/script.js"></script>
+   <script type="text/javascript">
+
+     let isMP3HasPlayed = false
+     document.onclick = (event) => {
+         if(!isMP3HasPlayed) {
+             document.querySelector('audio').play().then((x)=>console.info('sucess', x)).catch((e)=>console.error('fail', e))
+             hasPlayed = true
```
Comment (bokunodev) [inline app/Views/templates/default.php:240]: isMP3HasPlayed

File: app/Views/templates/default.php [hunk 4]
```diff
@@ -228,6 +231,24 @@ class="text-decoration-none">
...
+             document.onclick = null
+         }
+     }
+     document.ontouchstart = (event) => {
+         if(!isMP3HasPlayed) {
+             document.querySelector('audio').play().then((x)=>console.info('sucess', x)).catch((e)=>console.error('fail', e))
+             hasPlayed = true
```
Comment (bokunodev) [inline app/Views/templates/default.php:247]: isMP3HasPlayed

**Review by bokunodev — CHANGES_REQUESTED** [review]
isMP3HasPlayed

**Comment by irvanlabs** [comment]
Done

**Pull request merged**